src/styles/blocks/page.scss

```diff
@@ -0,0 +1,9 @@
+body {
+  margin: 0;
+  width: 100vw;
+  height: 100vh;
```

Comment:
Using 'height: 100vh;' can cause issues on mobile devices where the address bar is visible. Consider using 'min-height: 100vh;' to avoid this problem.